PanCardView/Resources/ResourcesInfo.cs

```diff
@@ -11,7 +11,7 @@ public static class ResourcesInfo
         private const string BlackRightArrowResourceName = "rightArrowBlack.png";
         private const string BlackLeftArrowResourceName = "leftArrowBlack.png";
 
-        private static Assembly ResourceAssembly => typeof(ResourcesInfo).Assembly;
+        private static Assembly ResourceAssembly => typeof(ResourcesInfo).GetTypeInfo().Assembly;
```

Comment:
```suggestion
        private static Assembly ResourceAssembly
            => typeof(ResourcesInfo).
#if !NETSTANDARD2_0
            GetTypeInfo().
#endif
            Assembly;
```

Reply:
I think it is not necessary. GetTypeInfo works fine for each target.